TicketingSystem.NotificationService/EmailService/EmailService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Mailjet.Client;
+using Mailjet.Client.TransactionalEmails;
+using TicketingSystem.Common.Model.DTOs.Other;
+
+namespace TicketingSystem.NotificationService.EmailService
+{
+    public class EmailService : IEmailService
+    {
+        private readonly IMailjetClient _mailjetClient;
+
+        public EmailService(IMailjetClient mailjetClient)
+        {
+            _mailjetClient = mailjetClient;
+        }
+
+        public async Task SendMail(Email email)
+        {
+            var transactionalEmail = new TransactionalEmailBuilder()
+                   .WithFrom(new SendContact("yurijsok@gmail.com"))
```

Comment:
I assume you should take the email address from the event payload.

Reply:
This is a sender address with is registered in mailjet. The receiver address comes from the payload (yes, they are the same)